.buildkite/pull-requests.json

```
@@ -0,0 +1,20 @@
+{
+    "jobs": [
+      {
+         "enabled": true,
+         "pipelineSlug": "elastic-agent-changelog-tool",
+         "allow_org_users": true,
+         "allowed_repo_permissions": ["admin", "write"],
+         "allowed_list": [ ],
+         "set_commit_status": false,
+         "build_on_commit": false,
```

As it is the main pipeline, I would set here these two fields as true

```suggestion
         "set_commit_status": true,
         "build_on_commit": true,
```